src/main/java/org/folio/entlinks/config/constants/Constrains.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.entlinks.config.constants;
+
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_ID;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_SOURCE_FILE_CODE;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_SOURCE_FILE_ID;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_SOURCE_FILE_NAME;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_SOURCE_FILE_SEQUENCE;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_AUTHORITY_SOURCE_FILE_URL;
+import static org.folio.entlinks.config.constants.ErrorCode.DUPLICATE_NOTE_TYPE_NAME;
+import static org.folio.entlinks.config.constants.ErrorCode.UNKNOWN_CONSTRAIN;
+import static org.folio.entlinks.config.constants.ErrorCode.VIOLATION_OF_RELATION_BETWEEN_AUTHORITY_AND_SOURCE_FILE;
+import static org.folio.entlinks.config.constants.ErrorCode.VIOLATION_OF_RELATION_BETWEEN_AUTHORITY_ARCHIVE_AND_SOURCE_FILE;
+
+import java.util.Map;
+
+public class Constrains {
```

Comment:
Maybe rename it to smth like DatabaseConstraintTranslator

Reply:
fixed